contracts/BlastYield.sol

```diff
@@ -46,7 +45,7 @@ contract BlastYield is OwnableTwoSteps {
      * @param wethReceiver The receiver of WETH.
      * @param usdbReceiver The receiver of USDB.
      */
-    function claim(address wethReceiver, address usdbReceiver) external virtual onlyOwner {
+    function _claim(address wethReceiver, address usdbReceiver) internal virtual {
```

Comment:
it just has to be internal, we are not overriding this function

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/e9cb33e1724f11ff061116c9d5cd6d8db4d1c452